grumphp.yml.dist

```diff
@@ -1,2 +1,22 @@
 imports:
-  - { resource: vendor/openeuropa/code-review/dist/drupal-conventions.yml }
\ No newline at end of file
+  - { resource: vendor/openeuropa/code-review/dist/drupal-conventions.yml }
+parameters:
+  tasks.phpcs.ignore_patterns:
+    - build/
+    - node_modules/
+    - vendor/
+  tasks.phpcs.triggered_by:
+    - inc
+    - install
+    - module
+    - php
+    - theme
+
+  extra_tasks:
+    phpparser:
+      ignore_patterns: %tasks.phpcs.ignore_patterns%
+      visitors:
+        declare_strict_types: ~
+      triggered_by: %tasks.phpcs.triggered_by%
+  extensions:
+    - OpenEuropa\CodeReview\ExtraTasksExtension
```

Comment:
Are we still using this in code-review 1.x? \cc @drupol

Reply:
Yes, that appears to be an outdated config file, it also does not check .module file, nor .php. let's should [use Drupal module template's one](https://github.com/openeuropa/drupal-module-template/blob/master/grumphp.yml.dist):

```yml
imports:
  - { resource: vendor/openeuropa/code-review/dist/drupal-conventions.yml }
parameters:
  tasks.phpcs.ignore_patterns:
    - build/
    - node_modules/
    - vendor/
  tasks.phpcs.triggered_by:
    - inc
    - install
    - module
    - php
    - theme
    - yml
```

Reply:
The parser is only on `dev-master` of **code-review**. so, we should keep the **phpparser**, until the next version of the **code-review**.